src/components/header-checkout/global-navigation-checkout.js

```diff
@@ -0,0 +1,99 @@
+/* Global Nav Functions */
```

Comment:
Pretty much the same as global-navigation.js since the mobile version here does have the hamburger menu icon.